karma.conf.js

```diff
@@ -164,6 +169,11 @@ module.exports = function(config) {
       args: ["--grep", config.grep || ""],
     },
   };
+
+  if (process.env.GITHUB_WORKFLOW) {
+    options.files.unshift("tests/use_bundled_profile.js");
+  }
+
   if (process.env.TRAVIS) {
```

Comment:
We don't need this anymore, right?

Reply:
Yep. Will remove in a cleanup (along with `detectBrowsers` - unless someone uses it)

Reply:
Maybe @marcoscaceres ?

Reply:
Should be ok to remove.

Reply:
Yay, we can finally remove mentioning of IE